### PR TITLE
Basic Windows Functionality Added

### PR DESCRIPTION
**Pull request by kjmcgrinder**

### CODE REVIEW

**Comment by gunesacar** [comment]
Thank you @kjmcgrinder. This has landed right before the holidays and then lost in the mailbox.

I've two additional request before merging this in:
- we need CI tests running on Windows. Travis apparently supports it.
- follow the naming conventions (`currentOS`->`current_os`)

File: tbselenium/tbdriver.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import shutil
 from os import environ, chdir
 from os.path import isdir, isfile, join, abspath
+from os import name as currentOS
```
Comment (gunesacar) [inline tbselenium/tbdriver.py]: this can be combined with line #3 (` from os import environ, chdir `)

**Comment by Land3r** [comment]
Is this feature planned to be merged ?
What needs to be done before it could be ? (aka Can I help ?)

**Comment by gunesacar** [comment]
> Is this feature planned to be merged ?
> What needs to be done before it could be ? (aka Can I help ?)

@Land3r I am willing to merge this PR if the issues listed in my comment and review are addressed:
https://github.com/webfp/tor-browser-selenium/pull/126#issuecomment-598436846
https://github.com/webfp/tor-browser-selenium/pull/126#pullrequestreview-373930853

Thanks for helping out.

File: tbselenium/tbdriver.py (hunk 1 above)
Comment (amoh-godwin) [reply tbselenium/tbdriver.py:4]: what you wanted to say is actually on line 2

**Comment by amoh-godwin** [comment]
@gunesacar how do I join this PR. I want to make the changes. Can you like accept his so I can make the changes? Or can you tell me how to join

**Comment by gunesacar** [comment]
Hi @amoh-godwin,
Thanks for your interest.
I believe you can check out the [kjmcgrinder/tor-browser-selenium branch](https://github.com/kjmcgrinder/tor-browser-selenium), commit your changes and send a new pull request.
That way we could preserve @kjmcgrinder's contributions.

**Comment by amoh-godwin** [comment]
@gunesacar can you please tell me the git code to get this going, please?

**Comment by amoh-godwin** [comment]
I have been successful in opening the pr in GitHub desktop by clicking the "Open with" button on top, but it seems I would need permission, either for this repo or for @kjmcgrinder repo. Please verify for me

**Comment by amoh-godwin** [comment]
Based on this [answer](https://stackoverflow.com/questions/13592653/multi-user-github-pull-requests/39963521#39963521), it appears I would need commit permission here before I can access his branch

**Comment by amoh-godwin** [comment]
Sorry for my above comments. I didn't read the phrase.
> send a new pull request.

**Pull request closed without merging**